datacube/drivers/postgres/_connections.py

```diff
@@ -180,7 +184,7 @@ def init(self, with_permissions=True):
         return is_new
 
     @contextmanager
-    def _connect(self):
+    def _connect(self) -> Iterator:
```

Comment:
Whilst strictly correct, this is not useful.   Should probably at least be `Iterator[_api.PostgresDbApi]`.

But again, imperfect but technically accurate typehints are better than no typehints.